src/Sheet.php

```diff
@@ -133,8 +133,10 @@ public function fromView(FromView $sheetExport)
 
         /** @var Html $reader */
         $reader = IOFactory::createReader('Html');
-        $reader->setSheetIndex($spreadsheet->getActiveSheetIndex());
+        // dump($spreadsheet->getActiveSheetIndex()); // Implementation bug: always return 0
+        $reader->setSheetIndex($spreadsheet->getSheetCount() - 1); // Insert content into the last sheet
```

Comment:
Can you add the comment above the line, instead of next to it? (Just our CS preference)